### PR TITLE
Separated into different groups and jobs list

### PR DESCRIPTION
**Pull request by monishaChengappa**

### CODE REVIEW

**Comment by swangful** [comment]
+1 :+1:

File: config.sample [hunk 1]
```diff
@@ -1,3 +0,0 @@
-u=user
-p=pass(API token recommended)
-s=site.domain.url
```
Comment (taoistmath) [inline config.sample]: Are we sure about checking in config.sample?

Comment (monishaChengappa) [inline config.sample]: .gitignore will ignore it so I don't have to change anything on my branch.

File: groupStatusesWriter.php [hunk 2]
```diff
@@ -211,7 +217,17 @@ function get_job_status($results, $jobName){
 
         foreach($block as $index => $value) {
             $valStat = get_job_status($jobs,$value);
-            $jobList .= "\t\t{\"name\": \"" . $value . "\", \"culprit\": \"" . get_status_culprit($value) . "\", \"status\": \"" . $valStat . "\", \"group\": \"" . $key . "\", \"subtabid\": \"". $count . $subcount .  "\"},\n";
+             
+            if($valStat == "failed" || $valStat == "run_fail") {
+                 $jobListRed .= "\t\t{\"name\": \"" . $value . "\", \"culprit\": \"" . get_status_culprit($value) . "\", \"status\": \"" . $valStat . "\", \"group\": \"" . $key . "\", \"subtabid\": \"". $count . $subcount .  "\"},\n";
+            }
+            else if($valStat == "passed" || $valStat == "run_pass") {
```
Comment (taoistmath) [inline groupStatusesWriter.php]: minor, but there is a method called elseif which should be used here.

http://stackoverflow.com/questions/3662412/are-elseif-and-else-if-completely-synonymous

File: groupStatusesWriter.php [hunk 3]
```diff
@@ -276,14 +292,25 @@ function get_job_status($results, $jobName){
                         break;
                 }
         }
-
-        $groups .= "\t\t{\"name\": \"" . $key . "\", \"status\": \"" . $statusArray[$worstJobStatus] . "\", \"count\": \"" . $count . "\"},\n";
+         
+        if($statusArray[$worstJobStatus] == "failed" || $statusArray[$worstJobStatus] == "run_fail") {
+            $groupRed .= "\t\t{\"name\": \"" . $key . "\", \"status\": \"" . $statusArray[$worstJobStatus] . "\", \"count\": \"" . $count . "\"},\n";
+        }
+        else if($statusArray[$worstJobStatus] == "passed" || $statusArray[$worstJobStatus] == "run_pass") {
```
Comment (taoistmath) [inline groupStatusesWriter.php]: same with elseif

File: config.sample (hunk 1 above)
Comment (dutchb) [inline config.sample]: Should not be deleting the sample from the repo. That's there to show others how to structure theirs.

**Comment by dutchb** [comment]
:+1:

**Pull request merged**